angular/src/lib/topbar/topbar-mobile.component.ts

```diff
@@ -64,7 +64,7 @@ export class TopbarMobileComponent {
   @HostListener('document:keydown', ['$event']) handleKeydown(
     e: KeyboardEvent
   ) {
-    if (e.code === 'Space' || e.code === 'Enter') {
+    if (e.code === 'Enter') {
```

Comment:
Per our discussion a while back...we know .code won't work in ie11 unless our polyfill is taking care of it.

https://caniuse.com/#search=.code